index.html

```diff
@@ -29,6 +29,13 @@ <h4>Name: Trinity</h4>
           <p>Quotes: "The Matrix isn't real." </p>
         </div>
 
+        <div class="character">
+          <h4>Name: Morpheus</h4>
+          <img src="/img/characters/Morpheus.jpg" alt="Morpheus" />
```

Comment:
The photo file is named "morpheus.jpg", but the `<img>` tag is "Morpheus.jpg".  Please correct it to be lowercase in the `<img>` tag.